tests/admin_views/tests.py

```diff
@@ -6181,3 +6182,46 @@ def test_implicitly_generated_pk(self):
         post_data = {'name': '2'}
         response = self.client.post(reverse('admin:admin_views_implicitlygeneratedpk_change', args=(1,)), post_data)
         self.assertEqual(response.status_code, 302)
+
+
+@override_settings(ROOT_URLCONF='admin_views.urls')
+class AddM2MFieldsToObjChangeHistoryTest(TestCase):
```

Comment:
Aren't many tests for log entries in admin_views module, but  test_change_view in admin_views/tests.py looks the best place to add this test.

 I could move it to the admin_utils module which has all the logentry tests, though it's not really a logentry test more a statement order check in view processing so doesn't feel quite right to me but I'll defer. I'd need to create a couple of models as there are none with m2m fields in that module but it's a small module, so maybe better than cluttering up admin_views?